### DISCUSSION TITLE
HTTP API request not working with some parameters like publishUser, publishPass, publishIPs

### DISCUSSION THREAD

**Discussion by thanhlnbka** [bluenviron/mediamtx#1330] [Questions]
I have tried running the server and editing the config with the http request. Some changes give good results but when I test with some parameters like pushlishUser, pushlishPass, pushlishIPs, it doesn't work as expected. Can you help me fix this? Thanks

**Comment by aler9** [comment]
Hello, i checked and in my case they worked perfectly, just make sure you're pointing to `http://localhost:9997/v1/config/paths/add/{name}` or `http://localhost:9997/v1/config/paths/edit/{name}`. Here's an example:

```
curl -v -d '{"source":"publisher","publishUser":"myuser","publishPass":"mypass"}'  http://localhost:9997/v1/config/paths/add/mypath
```

then

```
ffmpeg -stream_loop -1 -re -i file.mp4 -c copy -f rtsp rtsp://localhost:8554/mypath
```

the published will be blocked as expected since it didn't provide credentials.

[a comment by an automated account is not shown]
